packages/sit-onyx/src/components/OnyxBadge/OnyxBadge.vue

```diff
@@ -38,7 +38,7 @@ defineSlots<{
 </template>
 
 <style lang="scss">
-@use "../../styles/density.scss";
+@use "../../styles/mixins/density.scss";
```

Comment:
We also need to use the component layer here, right?

Reply:
true, done :)